package.json

```
@@ -37,12 +37,14 @@
   "contributors": [],
   "dependencies": {
     "@fast-csv/format": "^5.0.0",
+    "@idrinth-api-bench/typescript-language-from-yaml": "^1.4.0",
```

Biome (1.9.4)

[error] The key @idrinth-api-bench/typescript-language-from-yaml was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)